src/Base58.cpp

```diff
@@ -118,7 +119,7 @@ Data Base58::decode(const char* begin, const char* end) const {
     }
 
     // Skip trailing spaces.
-    it = std::find_if_not(it, end, std::isspace);
+    it = std::find_if_not(it, end, [](unsigned char ch) -> bool { return std::isspace(ch); });
```

Comment:
Is it a new requirement because of altered build flags?
I still see the old lines in upstream `master` - https://github.com/trustwallet/wallet-core/blob/master/src/Base58.cpp#L80